inference-engine/src/auto_plugin/auto_exec_network.cpp

```diff
@@ -21,10 +21,14 @@ AutoExecutableNetwork::AutoExecutableNetwork(const SoExecutableNetworkInternal&
 
 AutoExecutableNetwork::~AutoExecutableNetwork() = default;
 
-IInferRequestInternal::Ptr AutoExecutableNetwork::CreateInferRequestImpl(InputsDataMap networkInputs,
+InferenceEngine::IInferRequestInternal::Ptr AutoExecutableNetwork::CreateInferRequestImpl(InputsDataMap networkInputs,
                                                                          OutputsDataMap networkOutputs) {
-    SoIInferRequestInternal inferRequest = { _network, _network->CreateInferRequest() };
-    return std::make_shared<AutoInferRequest>(networkInputs, networkOutputs, inferRequest);
+    IE_THROW(NotImplemented);
+}
+
+InferenceEngine::IInferRequestInternal::Ptr AutoExecutableNetwork::CreateInferRequest() {
+    SoIInferRequestInternal inferRequest = {_network, _network->CreateInferRequest()};
+    return std::make_shared<AutoInferRequest>(_networkInputs, _networkOutputs, inferRequest);
```

Comment:
Based on this I think we need to implement only `CreateInferRequestImpl`
https://github.com/openvinotoolkit/openvino/blob/a5ea4fefbf83af2f23d5e54a8b9fabac3ae5a691/inference-engine/src/plugin_api/cpp_interfaces/impl/ie_executable_network_internal.hpp#L123-L139

It allows us not to call `setPointerToExecutableNetworkInternal` manually

Reply:
OK, as I understand this function sets _exeNetwork in this infer request, but it seems this _exeNetwork is not used at all.

Reply:
and I agree, we'd better implement `CreateInferRequestImpl`